src/Compilers/VisualBasic/Portable/Lowering/Rewriter.vb

```diff
@@ -14,6 +15,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             body As BoundBlock,
             previousSubmissionFields As SynthesizedSubmissionFields,
             compilationState As TypeCompilationState,
+            instrumentForDynamicAnalysis As Boolean,
+            ByRef dynamicAnalysisSpans As ImmutableArray(Of SourceSpan),
```

Comment:
Yes, it is an Out parameter. I hadn't noticed the existence of the <Out> attribute, even though it is already used in this very method.